introspection-engine/core/src/rpc.rs

```diff
@@ -95,14 +95,25 @@ impl RpcImpl {
 
     pub async fn introspect_internal(schema: String, force: bool) -> RpcResult<IntrospectionResultOutput> {
         let (config, url, connector) = RpcImpl::load_connector(&schema).await?;
+        let (config2, _, _) = RpcImpl::load_connector(&schema).await?;
```

Comment:
What we discussed — taking `&dyn Connector` into introspect could make this easier.